crates/sui-framework/packages/bridge/sources/bridge.move

```diff
@@ -274,24 +283,33 @@ module bridge::bridge {
         // TODO: why do we check validity of the route here? what if inconsistency?
         // Ensure route is valid
         // TODO: add unit tests
-        assert!(chain_ids::is_valid_route(source_chain, target_chain), EInvalidBridgeRoute);
-
+        let route = chain_ids::get_route(source_chain, target_chain);
         // get owner address
         let owner = address::from_bytes(message::token_target_address(&token_payload));
         // check token type
         assert!(treasury::token_id<T>() == message::token_type(&token_payload), EUnexpectedTokenType);
+        let amount = message::token_amount(&token_payload);
+        // Make sure transfer is within limit.
+        limiter::check_and_record_transfer<T>(clock, &mut inner.limiter, route, amount);
```

Comment:
for client's processing logic: can we return `option::none()` when limit is hit in this function, instead of abort?